app/controllers/prior_authority/searches_controller.rb

```diff
@@ -0,0 +1,36 @@
+module PriorAuthority
+  class SearchesController < BaseController
+    def show
+      @search = Search.new(search_params)
```

Comment:
so every other one of these is called `SearchForm`.. whould we do the same here?